### PR TITLE
Fun 122

### PR DESCRIPTION
**Pull request by flexi758**

### CODE REVIEW

File: application/layouts/scripts/sidr.phtml [hunk 1]
```diff
@@ -33,64 +39,175 @@ $params = Zend_Controller_Front::getInstance()->getRequest()->getParams();
...
+                            2.000.000
+                        </option>
                     </select>
                 </div>
             </ul>
         </nav>
-</div>
\ No newline at end of file
+</div>
+<script>
```
Comment (isaac-youwe) [inline application/layouts/scripts/sidr.phtml]: Are we going to throw all the Javascript in the view now? Move this function into .js file

File: application/layouts/scripts/sidr.phtml [hunk 2]
```diff
@@ -4,28 +4,34 @@
 <?php
 $params = Zend_Controller_Front::getInstance()->getRequest()->getParams();
 ?>
-<a id="filter" class="sidebar" href="#sidr" xmlns="http://www.w3.org/1999/html"><h4 class="vertical-text dev-mode">FILTERS</h4></a>
+<a id="filter" class="sidebar" href="#sidr" xmlns="http://www.w3.org/1999/html"><h4 class="vertical-text dev-mode">
+        FILTERS</h4></a>
 <div id="sidr">
-    <form action="/result">
+    <form id="sideform" action="/result">
```
Comment (isaac-youwe) [inline application/layouts/scripts/sidr.phtml]: change this sideform ID into something else, the whole form is being submitted not only the sideform/sidr

**Comment by isaac-youwe** [comment]
What about when you click on a filter without inserting a city name? I think this is not the right solution.
You should check if the search if filled before submit if search is not filled don't execute submit.

File: application/layouts/scripts/sidr.phtml [hunk 3]
```diff
@@ -33,64 +41,170 @@ $params = Zend_Controller_Front::getInstance()->getRequest()->getParams();
...
+                        <option value="0-1500000" class="filter_option"
+                                onclick="subm()" <?php echo in_array('0-1500000', $params) ? 'selected' : '' ?>>€ 0 - €
+                            1.250.000
+                        </option>
+                        <option value="0-2000000" class="filter_option"
+                                onclick="subm()" <?php echo in_array('0-2000000', $params) ? 'selected' : '' ?>>€ 0 - €
+                            1.500.000
+                        </option>
+                        <option value="0-6000000" class="filter_option"
+                                onclick="subm()" <?php echo in_array('0-6000000', $params) ? 'selected' : '' ?>>€ 0 - €
```
Comment (isaac-youwe) [inline application/layouts/scripts/sidr.phtml]: please change subm() into submitForm() onoverzichtelijk @flexi758

File: public/js/submitfilter.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by serhildan on 10-6-15.
+ */
+
+function subm() {
+    if (document.getElementById('searchid').value == 0) {
+        alert("Vul een buurt of plaats in aub");
```
Comment (isaac-youwe) [inline public/js/submitfilter.js]: I don't recommend the alert window:
1. it's irritating
2. a user is able to choose filters before typing de buurt name

I would recommend something like this:
```
function submitForm() {
  if (document.getElementById('searchid').value.length => 3) {
   document.getElementById('form').submit();
}
}
}
```

File: public/js/submitfilter.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+/**
```
Comment (isaac-youwe) [inline public/js/submitfilter.js]: I see there is a filter.js you should use that one instead of creating a new one. let's stay organised ;)

**Comment by flexi758** [comment]
@isaac-youwe created a bit different 'if loop' because the one u wanted didn't work

**Comment by isaac-youwe** [comment]
It was an example

**Comment by flexi758** [comment]
@isaac-youwe the other way around, so without a empty if loop it didn't work

**Comment by isaac-youwe** [comment]
UPDATE YOUR BRANCH 

We can’t automatically merge this pull request.

@flexi758

**Pull request merged**